android-components/components/ui/tabcounter/src/main/res/layout/mozac_ui_tabcounter_layout.xml

```diff
@@ -39,7 +39,7 @@
 
         <androidx.appcompat.widget.AppCompatImageView
             android:id="@+id/counter_mask"
-            app:srcCompat="@drawable/mozac_ui_tabcounter_badge"
+            app:srcCompat="@drawable/mozac_ic_private_mode_circle_fill_20"
```

Comment:
Did we need to tint this icon? The oval was `photonPurple60` before in [2a7842b54bffL8](https://github.com/mozilla-mobile/firefox-android/pull/3847/files#diff-ac479197d8ac6efe0395c7d9d2d4fd04f509cdbbf76897d6cefe2a7842b54bffL8)

Reply:
added `photonPurple60` tint